lib/sync-DataSetModel.js

```diff
@@ -156,6 +159,18 @@ var self = {
     });
   },
 
+  doResponseInterceptor: function (dataset_id, params, cb) {
+    self.getDataset(dataset_id, function (err, dataset) {
+      if (err) return cb(err);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)